cylc/uiserver/websockets/tornado.py

```diff
@@ -130,24 +131,36 @@ async def on_start(self, connection_context, op_id, params):
 
         params['root_value'] = op_id
         execution_result = self.execute(params)
+        try:
+            if isawaitable(execution_result):
+                execution_result = await execution_result
+            if not hasattr(execution_result, '__aiter__'):
+                await self.send_execution_result(
+                    connection_context, op_id, execution_result)
+            else:
+                iterator = await execution_result.__aiter__()
+                connection_context.register_operation(op_id, iterator)
+                async for single_result in iterator:
+                    if not connection_context.has_operation(op_id):
+                        break
+                    await self.send_execution_result(
+                        connection_context, op_id, single_result)
+        except Exception as e:
+            await self.send_error(connection_context, op_id, e)
+        await self.send_message(connection_context, op_id, GQL_COMPLETE)
+        await connection_context.unsubscribe(op_id)
+        await self.on_operation_complete(connection_context, op_id)
```

Comment:
This line is not being called, even if moved higher up in the code, or placed in a `finally` block. This results in the dictionary not getting cleaned correctly, which is not ideal.

Reply:
I can't replicate this issue. Will investigate if we can replicate it, otherwise safe to ignore for now.